init/common.js

```diff
@@ -189,6 +197,18 @@ const createProject = async (interestId, userId) => {
   });
 };
 
+const createProjectWithoutTeam = async (interestId, userId) => {
```

Comment:
You could modify previous method `createProject` and pass project object and boolean flag `addTeam`